integration-tests/google-pubsub/README.adoc

```diff
@@ -0,0 +1,21 @@
+== Google Pubsub integration tests
+
+=== Local container
+
+Google emulator differs a little bit from the real account.  For example there is a test which passes on local emulator but fails on real account (see an https://issues.apache.org/jira/browse/CAMEL-18277[issue] in Camel project).
+
+Tests are executed with local container, unless `GOOGLE_APPLICATION_CREDENTIALS` and `GOOGLE_PROJECT_ID` properties (both) are set.
```

Comment:
Neat: avoiding parenthesis could make it much clearer ? Like `unless both A and B properties are set` ?

Reply:
done